tools/dump_all_tables_data_only.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+GIT_MYSQL=/[BACKUP_LOCATION]
+for T in `mysql -N -B -e 'show tables'`;
+do
+    echo "Backing up $T"
+    mysqldump Demo --complete-insert --no-create-db --no-create-info --skip-add-drop-table --verbose $T | sed -r 's/INSERT INTO (`[^`]+`)/TRUNCATE TABLE \1; INSERT INTO \1/g' > /data/demo_database_backups/split/demo-19-1_$T.sql
```

Comment:
I think this is supposed to be `sed -e`.. `-r` doesn't show up in the man page.

Reply:
seems like `-r` is the old way of doing `-e`

```
'-r'
'--regexp-extended'
     Use extended regular expressions rather than basic regular
     expressions.  Extended regexps are those that 'egrep' accepts; they
     can be clearer because they usually have fewer backslashes.
     Historically this was a GNU extension, but the '-E' extension has
     since been added to the POSIX standard
     (http://austingroupbugs.net/view.php?id=528), so use '-E' for
     portability.  GNU sed has accepted '-E' as an undocumented option
     for years, and *BSD seds have accepted '-E' for years as well, but
     scripts that use '-E' might not port to other older systems.  *Note
     Extended regular expressions: ERE syntax.
```